queries.sql

```diff
@@ -9,3 +9,44 @@
  SELECT * FROM animals WHERE neutered = true;
  SELECT * FROM animals WHERE name <> 'Gabumon';
  SELECT * FROM animals WHERE  weight_kg BETWEEN 10.4 and 17.3;
+
+ SELECT * FROM animals;
+ ALTER TABLE animals ADD COLUMN species VARCHAR(200);
+ START TRANSACTION;
+ UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+ UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+ SELECT * FROM animals;
+ DELETE FROM animals;
+ SELECT * FROM animals;
+  ROLLBACK;
+ COMMIT;
+ SELECT * FROM animals;
+ START TRANSACTION;
+ DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+ SAVEPOINT gdm;
+ UPDATE animals SET weight_kg = weight_kg * -1;
+ ROLLBACK TO SAVEPOINT gdm;
+ SELECT * FROM animals;
+ UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+ SELECT * FROM animals;
+ COMMIT;
+ SELECT COUNT(*) AS total_animals FROM animals;
+ SELECT COUNT(*) AS never_escaped_animals FROM animals WHERE escape_attempts = 0;
+ SELECT COUNT(*) AS never_escape_animals From animals WHERE escape_attempts = 0;
```

Comment:
- [ ] You're repeating the same query here, Please make sure to delete the duplicate query here.